platform/kube/inject/testdata/auth.yaml.injected

```diff
@@ -52,6 +52,8 @@ spec:
         - ""
         - --proxyAdminPort
         - "15000"
+        - --controlPlaneAuthPolicy
+        - "0"
```

Comment:
yeah. definitely move to either simple --enable-con... or use text strings.

Reply:
changed to text based now the command line is
--controlPlaneAuthPolicy NONE
or
--controlPlaneAuthPolicy MUTUAL_TLS